boefjes/boefjes/plugins/kat_nmap_ip_range/schema.json

```diff
@@ -5,16 +5,30 @@
         "TOP_PORTS_TCP": {
             "title": "TOP_PORTS_TCP",
             "type": "integer",
-            "minimum": 1,
+            "minimum": 0,
             "maximum": 65535,
             "description": "Scan TOP_PORTS_TCP most common TCP ports. Defaults to 250."
         },
         "TOP_PORTS_UDP": {
             "title": "TOP_PORTS_UDP",
-          "type": "integer",
-          "minimum": 1,
-          "maximum": 65535,
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 65535,
             "description": "Scan TOP_PORTS_UDP most common UDP ports. Defaults to 10."
+        },
+        "MIN_VLSM_IPV4": {
+            "title": "MIN_VLSM_IPV4",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 32,
+            "description": "Minimum variable-length subnet mask for IPv4-ranges. Defaults to 0. Use this value to prevent scanning large ranges."
+        },
+        "MIN_VLSM_IPV6": {
+            "title": "MIN_VLSM_IPV6",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 128,
+            "description": "Minimum variable-length subnet mask for IPv6-ranges. Defaults to 0. Use this value to prevent scanning large ranges."
```

Comment:
maybe we should introduce reasonable defaults. a /118 subnet for ipv6 seems manageable, and a /22 for ipv4.
Both are equal to 1024 addresses.

Reply:
It is possible to show a default value for a setting in the interface? Otherwise it would not be clear to an end-user what the default is, resulting in missed scans if we modify it here

Reply:
https://json-schema.org/understanding-json-schema/reference/generic.html#annotations
You could use the 'default' value keyword. If we don't render that, we should.

Reply:
Added, but it indeed seems like its not rendered

Reply:
https://github.com/minvws/nl-kat-coordination/pull/1808 implements the default rendering in the forms.

Reply:
This form however is not a jsonschema form just yet. 
https://github.com/minvws/nl-kat-coordination/issues/447